build.gradle.kts

```diff
@@ -22,19 +25,25 @@ java {
 
 val sezpoz = "net.java.sezpoz:sezpoz:1.13"
 
+configurations.all {
+    exclude("org.jenkins-ci", "annotation-indexer")
```

Comment:
It would be better if we could just use repo.jenkins-ci.org and not exclude this, setting up permissions and publishing there doesn't seem to be straightforward, and it's unreliably from what I hear.

Reply:
Thanks for the ping.

I'm hesitant to introduce a new buildscript repository requirement because I can't add that from within the plugin, so the plugin would no longer work out of the box. It may be possible to resolve these dependencies from another configuration only when the task executes. I'll take a look at what our options are.

Reply:
Updated the PR to use the [Worker API](https://docs.gradle.org/6.7.1/userguide/custom_tasks.html#worker_api) with classloader isolation and this should work. `org.jenkins-ci:annotation-indexer` is no longer excluded.